src/Desktop/UIAutomation/CustomObjects/Registrar.cs

```diff
@@ -63,7 +65,9 @@ private static UIAutomationType GetUnderlyingUIAType(CustomUIAPropertyType type)
             switch (type)
             {
                 case CustomUIAPropertyType.String: return UIAutomationType.UIAutomationType_String;
-                case CustomUIAPropertyType.Int: return UIAutomationType.UIAutomationType_Int;
+                case CustomUIAPropertyType.Int:
+                case CustomUIAPropertyType.Enum:
```

Comment:
The fallthrough here seems odd. From a code readability perspective, it might make sense to duplicate the `return UIAutomationType.UIAutomationType_Int` on 2 lines. The JIT optimizer is smart enough to combine the cases in release mode